infinite_tracing/lib/infinite_tracing/worker.rb

```diff
@@ -52,7 +52,7 @@ def stop
 
       def start_thread
         NewRelic::Agent.logger.debug("starting worker #{@name} thread...")
-        @worker_thread = Thread.new do
+        @worker_thread = NewRelic::Agent::Threading::AgentThread.create("infinite_tracing_worker") do
```

Comment:
Could 8T use a new test too?

Reply:
added in e76793206b045c5a16d3295891aa4f783916e47b